docs/pull_request_template.md

```diff
@@ -1,10 +1,10 @@
 [EVG-<number>](https://jira.mongodb.org/browse/EVG-<number>)
 
-### Description 
+### Description
 < add description, context, thought process, etc >
```

Comment:
Can these be replaced with comments so if someone forgets to change these they don't show up

Reply:
controversial take: I think that it's good if forgetting to update it obvious. That way it's easy to spot and fix.

Reply:
I don't have strong feelings one way or the other here. I'm inclined to leave it as is since I'm not the one who came up with the template.